### PR TITLE
Issue #27 Menu expansion

### PR DESCRIPTION
**Pull request by Alex-Hayeur**
Issue #27, Profile page now prompts user to log in or sign up if logged out. Modified the settings menu layout and added buttons for notification settings and app information

### CODE REVIEW

File: app/src/main/java/Fragments/SettingsFragment.java [hunk 1]
```diff
@@ -52,18 +54,31 @@ public void onStart() {
...
+        notificationsB = (Button)getView().findViewById(R.id.notificationsB);
+
+
+
 
         // SET Auth State Listener
         authListener = new FirebaseAuth.AuthStateListener() {
             @Override
             public void onAuthStateChanged(@NonNull FirebaseAuth firebaseAuth) {
                 if(authRef.getCurrentUser() != null) {
```
Comment (kkhh001) [inline app/src/main/java/Fragments/SettingsFragment.java:69]: Minor tweak suggestion: I would add for the if statement : user logged in 
and for the else statement: user logged out, just for reference purposes!

**Comment by EvanMateo7** [comment]
I will be refactoring the authentication state listener for the fragments in the future.

**Review by kkhh001 — APPROVED** [review]
Good use of the naming convention. Played around with the app: nice touch with the little pop up messages to notify the user if signed in or signed out. Overall 👍

File: app/src/main/java/Fragments/SettingsFragment.java (hunk 1 above)
Comment (davidberard2) [reply app/src/main/java/Fragments/SettingsFragment.java:69]: Can this be implemented in a future sprint???

**Comment by davidberard2** [comment]
-I would suggest a redefinition of necessity for the upcoming sprint, I think it would be useful to sit with Daniel to determine the need to adapt for item addition. 
-The profile log out is really nice and pretty useful, it will allow to redirect to purpose of the setting fragment.

**Review by davidberard2 — COMMENTED** [review]
The overall code and purpose is good. As I said, I would look into redirecting our efforts to make each minute spend coding a little more productive in terms of goal and objective.

**Comment by LHuss** [comment]
Travis : 
/home/travis/build/davidberard2/SOEN341GROUPC/app/build/intermediates/res/merged/debug/layout/fragment_settings.xml:43: error: Error parsing XML: duplicate attribute

**Pull request merged**